codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/localservice/nameresolver/SmithyNameResolver.java

```diff
@@ -60,9 +62,14 @@ public static String shapeNamespace(final Shape shape) {
   }
 
   public static String smithyTypesNamespace(final Shape shape) {
```

Comment:
The type namespace for service should be "github.com/aws/aws-sdk-go-v2/service/dynamodb/types" (or KMS one) but there can be multiple service in a model and this will cause confusion between kms and ddb. So, I have decided to make it kmstypes  or dynamodbtypes.

Reply:
Yes - we always need to alias it.